$OpenBSD: patch-dirmngr_ks-engine-hkp_c,v 1.1 2015/09/29 20:51:42 edd Exp $

Fix use of pointer which may have moved due to realloc().
https://bugs.gnupg.org/gnupg/issue2107

--- dirmngr/ks-engine-hkp.c.orig	Tue Sep 29 15:05:02 2015
+++ dirmngr/ks-engine-hkp.c	Tue Sep 29 15:05:26 2015
@@ -512,7 +512,7 @@ map_host (ctrl_t ctrl, const char *name, int force_res
               xfree (reftbl);
               return err;
             }
-          qsort (reftbl, refidx, sizeof *reftbl, sort_hostpool);
+          qsort (hi->pool, refidx, sizeof *reftbl, sort_hostpool);
         }
       else
         xfree (reftbl);
